controlplane/kubeadm/config/webhook/manifests.yaml

```diff
@@ -98,3 +98,25 @@ webhooks:
     resources:
     - kubeadmcontrolplanetemplates
   sideEffects: None
+- admissionReviewVersions:
+  - v1
+  - v1beta1
+  clientConfig:
+    service:
+      name: webhook-service
+      namespace: system
+      path: /validate-scale-controlplane-cluster-x-k8s-io-v1beta1-kubeadmcontrolplane
+  failurePolicy: Fail
+  matchPolicy: Equivalent
+  name: validation-scale.kubeadmcontrolplane.controlplane.cluster.x-k8s.io
+  rules:
+  - apiGroups:
+    - controlplane.cluster.x-k8s.io
+    apiVersions:
+    - v1beta1
+    operations:
+    - CREATE
```

Comment:
nit: we might only need one of those operations (I think you can find out via `kubectl scale .. -v=8`

Reply:
It's a `PATCH` Operation, so we can only keep `UPDATE`. Will update it.